app/lib/frontend/templates/views/pkg/screenshots.dart

```diff
@@ -60,11 +63,15 @@ d.Node collectionsIcon() {
 }
 
 d.Node screenshotThumbnailNode(
-    String thumbnailUrl, List<String>? screenshotUrls) {
+  String thumbnailUrl,
+  List<String>? screenshotUrls,
+  List<String>? screenshotDescriptions,
+) {
   return d.div(classes: [
     'thumbnail-container'
   ], attributes: {
     'data-thumbnail': screenshotUrls!.join(','),
+    'data-thumbnail-descriptions': screenshotDescriptions!.join(','),
```

Comment:
what happens if a description has a comma in it?

Reply:
Maybe this should be a `data-thumbnail-description-N` attribute, or something json-encoded?

Reply:
done!